tests/system/action/base.py

```diff
@@ -303,7 +308,7 @@ def set_user_groups(self, user_id: int, group_ids: List[int]) -> None:
         )
         meeting_users_new = {
             meeting_id: {
-                "id": last_meeting_user_id + 1,
+                "id": (last_meeting_user_id := last_meeting_user_id + 1),  # noqa: F841
```

Comment:
same change as in a former PR, you can remove the assigment and the `noqa`

Reply:
I'm unable to recreate the error, can you test if mypy runs without the `noqa`?

Reply:
And then, it plopped up again;-)